### DISCUSSION TITLE
Thank you, everyone!

### DISCUSSION THREAD

**Discussion by bitbound** [immense/Remotely#262] [General]
I just wanted to take a moment to say thank you! :) I've been getting a lot of amazing support and feedback lately, and it's been extremely motivating.

Your support is very much appreciated! Thanks!

**Comment by stevesobol** [comment]
No, we should be thanking you. You've spent an insane amount of time developing an amazing product. We appreciate you.

**Comment by iamk3** [comment]
I discovered Remotely from Awesome Open Source on YouTube. I love this product! You have spent lots of time creating an amazing and superior software product! Thank you so very much.

**Comment by opensource008** [comment]
We are the one need to say thank you for your wonderful work,
